clarifai/runners/dockerfile_template/Dockerfile.cuda.template

```diff
@@ -57,54 +42,28 @@ RUN python -m pip install clarifai
 # Finally copy everything we built into a distroless image for runtime.
 ######################>#######
 ARG TARGET_PLATFORM=linux/amd64
-# FROM --platform=$TARGET_PLATFORM gcr.io/distroless/python3-debian12:latest
-FROM --platform=$TARGET_PLATFORM gcr.io/distroless/python3-debian12:debug
+FROM --platform=$TARGET_PLATFORM gcr.io/distroless/python3-debian12:latest
+# FROM --platform=$TARGET_PLATFORM gcr.io/distroless/python3-debian12:debug
 ARG PYTHON_VERSION=${PYTHON_VERSION}
 # needed to call pip directly
 COPY --from=build /bin/sh /bin/sh
 
 # Copy driver libraries based on architecture
-# Set FOLDER_NAME based on TARGET_PLATFORM
 ENV TARGET_PLATFORM=${TARGET_PLATFORM}
 ENV FOLDER_NAME="x86_64-linux-gnu"
```

Comment:
should use `*-linux-gnu`

Reply:
*where globbing is possible like for `COPY`

Reply:
oh removed this section as those vars aren't used.